mipdb/usecases.py

```diff
@@ -317,4 +316,188 @@ def update_actions_on_dataset_deletion(record, conn):
     action_record = dict()
     action_record["action_id"] = actions_table.get_next_id(conn)
     action_record["action"] = json.dumps(record)
-    actions_table.insert_values(action_record, conn)
\ No newline at end of file
+    actions_table.insert_values(action_record, conn)
+
+
+class EnableSchema(UseCase):
+    def __init__(self, db: DataBase) -> None:
+        self.db = db
+
+    def execute(self, name, version) -> None:
+        metadata = Schema(METADATA_SCHEMA)
+        schemas_table = SchemasTable(schema=metadata)
+
+        with self.db.begin() as conn:
+            schema_id = self._get_schema_id(name, version, conn)
+            schemas_table.set_status_schema("ENABLED", schema_id, conn)
```

Comment:
I think it's important when enabling or disabling a schema/dataset to verify than it was previously disabled or enabled and emit a warning otherwise.